workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/types.d.ts

```diff
@@ -570,6 +571,13 @@ type ProductSpecification = {
 	value: {[key: string]: string};
 };
 
+type TierPrice = {
+	currency: string;
+	price: number;
+	priceFormatted: string;
+	quantity: number;
+};
+
```

Comment:
We already have TierPrices type in `/src/components/LicensePriceCard/LicensePriceChildren.tsx`